src/seedu/StudentHandBook/commands/RandomCommand.java

```diff
@@ -0,0 +1,55 @@
+package seedu.StudentHandBook.commands;
+
+import seedu.StudentHandBook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+import java.util.*;
+
+
+public class RandomCommand extends Command {
+
+    public static final String COMMAND_WORD = "random";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n"
+            + "Display a random person and their details in the address book. \n\t"
+            + "Example: " + COMMAND_WORD;
+
+
+    @Override
+
+    public CommandResult execute() {
+        final List<ReadOnlyPerson> randomPerson = getRandomPerson();
+        if(randomPerson.size() == 0) {
+            return new CommandResult(getMessageForRandomFail());
+        }
+        else {
+            return new CommandResult(getMessageForRandomShown(), randomPerson);
+        }
+    }
+
+
+    private List<ReadOnlyPerson> getRandomPerson() {
+        final List<ReadOnlyPerson> randomEntry = new ArrayList<>();
```

Comment:
If you are interested in finding only one random person, then there is no need for creating a list.